### PR TITLE
Update README and License

### PR DESCRIPTION
**Pull request by juandavidkincaid**
[CE-598](https://constellationnetwork.atlassian.net/browse/CE-598)

### Changes
+ Updated readme, license, and package.json => added info.

[CE-598]: https://constellationnetwork.atlassian.net/browse/CE-598?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,60 @@
-# Getting Started with Create React App
+# DAG Explorer Project (v2)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+DAG Explorer is an open-source tool designed for the Constellation Network to monitor transaction statuses and other important network information. This project is built with TypeScript and React.
```
Comment (AlexBrandes) [inline README.md:3]: The only thing I would change here is linking to the live site so it's clear what the relationship is.

File: README.md [hunk 2]
```diff
@@ -1,46 +1,60 @@
...
 
-In the project directory, you can run:
+- Monitor transaction statuses
+- View network snapshots
+- Access wallet addresses
+- Supports MainNet and test networks
+- Built with TypeScript and React for a robust and scalable application
 
-### `yarn start`
+## Installation
```
Comment (AlexBrandes) [inline README.md:13]: There are API dependencies on the Lattice API, aren't there? Can we explain those somewhere here?

**Review by AlexBrandes — APPROVED** [review]
I added a couple comments but this looks good overall. Thanks for adding the license too.

File: README.md (hunk 2 above)
Comment (juandavidkincaid) [reply README.md:13]: changing...

**Pull request merged**